README.md

```diff
@@ -1,12 +1,68 @@
 # erddaplogs
 
+Quick utilities for parsing nginx and apache logs.
+
 Try it out on Binder [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/callumrollo/erddaplogs/HEAD?labpath=weblogs-parse-demo.ipynb)
 
-Quick utilities for parsing nginx and apache logs.
+This package takes apache and/or nginx logs as input. It is made to analyse visitors to an ERDDAP server, but should work on any web traffic.
+
+### Installation
+* #### From the repo, using conda/mamba:
+
+First, clone the repo:
+```sh 
+git clone https://github.com/callumrollo/erddaplogs.git
+```
+then you can use Conda:
+```sh
+conda env create --file environment.yml # create the environment 
+conda activate erddaplogs # activate the environment
+```
+or Mamba:  
```

Comment:
I don't think the mamba instructions are necessary, those using mamba will know to substitute it for conda